crates/forge/src/main.rs

```diff
@@ -148,6 +166,16 @@ fn test_workspace(args: TestArgs) -> Result<bool> {
     }
 
     let scarb_metadata = MetadataCommand::new().inherit_stderr().exec()?;
+
+    let snforge_std_version_requirement = snforge_std_version_requirement();
+    if !package_matches_version_requirement(
+        &scarb_metadata,
+        "snforge_std",
+        &snforge_std_version_requirement,
+    )? {
+        print_warning(&anyhow!("Package forge_std version does not meet the recommended version requirement {snforge_std_version_requirement}, it might result in unexpected behaviour"));
```

Comment:
```suggestion
        print_warning(&anyhow!("Package snforge_std version does not meet the recommended version requirement {snforge_std_version_requirement}, it might result in unexpected behaviour"));
```
Why without sn prefix?

Reply:
Done